### DISCUSSION TITLE
When deploying LibreChat to a read-only system by default, which paths should be made writable?

### DISCUSSION THREAD

**Discussion by erickrf** [danny-avila/LibreChat#4032] [Troubleshooting] [answered]
I'm deploying LibreChat to a kubernetes cluster, which has a read-only filesystem by default. I can mount volumes to specific paths that allow writing, and it would be great to know them in advance. Via trial and error, I identified so far:

- `/app/api/logs`
- `/app/data`

I would guess `/app/client/public/images` as well if you work with images. So, to avoid unpleasant surprises and help others doing the same, is there any other path to be considered?

**Comment by fuegovic** [comment, marked as answer]
On our HF demo we have these:
```
/app/uploads/temp
/app/client/public/images
/app/api/logs/
/app/data

```
